include/dlt/dlt_common.h

```diff
@@ -111,6 +111,16 @@
 #      define DLT_PACKED
 #   endif
 
+#   ifdef DLT_DAEMON_USE_QNX_MESSAGE_IPC
+#      ifdef __QNX__
```

Comment:
As commented above, we don't require this ifdef, no error message at line 120 either. The same for other places.